### PR TITLE
[master] Sort TSV

### PR DESCRIPTION
**Pull request by mdouchement**

### CODE REVIEW

File: scanner.go [hunk 1]
```diff
@@ -28,28 +28,40 @@ import (
...
 )
 
+// Scanner conatins all stuff for reading a buffered file
 type Scanner struct {
 	f       *os.File      // The file provided by the client.
 	r       *bufio.Reader // Buffered reader on given file.
 	keepnls bool          // Keep the newline sequence in returned strings
 	token   []byte        // Last token returned by split (scan).
 	err     error         // Sticky error.
+	seekers []seeker      // Internal use
```
Comment (dehau) [inline scanner.go]: seekers => _seekers for private property ?

**Comment by mdouchement** [comment]
When a field  (or func/method) starts with a capital letter, it means that the field is public. With a lower case letter, it means that the field is private.

File: scanner.go (hunk 1 above)
Comment (mdouchement) [inline scanner.go]: When a field  (or func/method) starts with a capital letter, it means that the field is public. With a lower case letter, it means that the field is private.

Comment (dehau) [inline scanner.go]: Nice :+1:

**Pull request merged**